tests/unit/data/metadata.json

```diff
@@ -28,7 +28,7 @@
     "data_license": "CC BY 4.0",
     "data_license_url": "https://creativecommons.org/licenses/by/4.0/legalcode",
     "version": "20.09d",
-    "data_url": "https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/archive/20.09d_Release/",
+    "data_url": "https://evs.nci.nih.gov/ftp1/NCI_Thesaurus/archive/2020/20.09d_Release/",
```

Comment:
💀 💀 💀 💀 💀

Reply:
Who would've known it'd change once 2021 came?!